Source/JavaScriptCore/dfg/DFGByteCodeParser.cpp

```diff
@@ -25,6 +25,7 @@
 
 #include "config.h"
 #include "DFGByteCodeParser.h"
+#include "interpreter/CallFrame.h"
```

Comment:
This can just be `#include "CallFrame.h"`

Reply:
Updated